pkg/handler/handler.go

```diff
@@ -203,6 +203,9 @@ func (o *OpenAPIService) RegisterOpenAPIVersionedService(servePath string, handl
 							return
 						}
 					}
+					contentType := accepts.Type + "/" + accepts.SubType
+					w.Header().Set("Content-Type", contentType)
```

Comment:
when this got pulled into kubernetes, we're seeing this error:

> Error from server (InternalError): Internal error occurred: mime: unexpected content after media subtype

Reply:
failed in https://github.com/kubernetes/kubernetes/pull/114869

Reply:
not sure if this PR or https://github.com/kubernetes/kube-openapi/pull/334/files was the cause... both were in the diff pulled in in that PR